### PR TITLE
Finalize shop UI, routing and templates

### PR DESCRIPTION
**Pull request by pjedrzejewski**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | yes |
| BC breaks?      | yes |
| Related tickets | - |
| License         | MIT |

* Also changes the payment method change flow and routing.
* Removed "sylius_get_cart()" helper from Twig, which was just plain wrong.
* WIP

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Gulpfile.js [hunk 1]
```diff
@@ -75,6 +77,8 @@ gulp.task('shop-css', function() {
 });
 
 gulp.task('shop-img', function() {
+    gulp.src(['../../../../node_modules/lightbox2/dist/images/*']).pipe(gulp.dest(shopRootPath + 'images/'));
```
Comment (Arminek) [inline src/Sylius/Bundle/ShopBundle/Gulpfile.js]: You should move this to `img: [node_modules + '/lightbox2/dist/images/*']`.

File: features/checkout/shipping_order/preventing_shipping_step_completion_without_a_selected_method.feature [hunk 2]
```diff
@@ -31,7 +31,7 @@ Feature: Preventing shipping step completion without a selected method
         Then I should not be able to complete the shipping step
         And there should be information about no shipping methods available for my shipping address
       
-    @ui
+    @ui @todo
```
Comment (michalmarcinkowski) [inline features/checkout/shipping_order/preventing_shipping_step_completion_without_a_selected_method.feature]: Why these are `@todo`?

Comment (michalmarcinkowski) [inline features/checkout/shipping_order/preventing_shipping_step_completion_without_a_selected_method.feature]: But it looks like already fixed https://github.com/Sylius/Sylius/pull/6843/files/85aa2899973df5748324da547e568e247ce0a630#diff-81e9307596c4fe5ab65cbb5649572290R1.

Comment (michalmarcinkowski) [inline features/checkout/shipping_order/preventing_shipping_step_completion_without_a_selected_method.feature]: Ok, it doesn't handle the case when no methods are available.

**Comment by michalmarcinkowski** [comment]
Awesome job! Thanks Paweł 👍

**Pull request merged**